checker/src/org/checkerframework/checker/propkey/PropertyKeyAnnotatedTypeFactory.java

```diff
@@ -52,6 +54,15 @@ public PropertyKeyAnnotatedTypeFactory(BaseTypeChecker checker) {
     }
 
     @Override
+    protected Set<Class<? extends Annotation>> createSupportedTypeQualifiers() {
+        // TODO: it is possible to load @PropertyKey and @UnknownPropertyKey using reflection, but it would also load @PropertyKeyBottom
```

Comment:
Why is this checker using Bottom instead of PropertyKeyBottom?
The shared Bottom should only be used in the Subtyping Checker.
Why have the dedicated PropertyKeyBottom and then not use it?